Kernel/Bus/VirtIO/Device.cpp

```diff
@@ -27,7 +27,7 @@ UNMAP_AFTER_INIT ErrorOr<void> Device::initialize_virtio_resources()
 }
 
 UNMAP_AFTER_INIT VirtIO::Device::Device(NonnullOwnPtr<TransportEntity> transport_entity)
-    : m_class_name(transport_entity->determine_device_class_name())
+    : m_class_name(transport_entity -> determine_device_class_name())
```

Comment:
🤔 there's exactly one instance of this I can see, maybe it'd go away if we put the thing in another set of parens?
not a blocker obv, just thinking out loud.

Reply:
There's another in Kernel/Devices/GPU/Console/VGATextModeConsole.cpp
Parens don't fix it unfortunately.

Reply:
Huh. alrighty then.